CondTools/SiPixel/plugins/BuildFile.xml

```diff
@@ -1,4 +1,14 @@
+<use name="boost"/>
```

Comment:
Did you check if any of these lines, moved here from test\BuildFile.xml, can be now removed from the original BuildFile?

Reply:
Now, I did. There is some stuff that's not actually needed in this file (commit comes soon), but the original BuildFile actually deals with much more broader Analyzers